downstream/README.md

```diff
@@ -0,0 +1,238 @@
+
+- Version in specfile is updated and there is a new changelog entry.
+- New version of source is saved in lookaside cache and the spec-file uses that new version.
+- Some files can be directly synced to dist-git.
+
+#### How to enable?
+
+- I can install the Packit GitHub application and configure the `propose-downstream` job. (already possible)
+- I can do this without the upstream's permission:
+  - Send a pull-request to a repository containing the list of repositories to sync.
+- I can configure this in dist-git.
+  - I can use packit config to enable this.
+
+#### How to trigger?
+
+As a maintainer, I expect this to be done when there is a new upstream release. Also, I have a way to trigger this manually when I need and I have a way to re-create the PRs when there is a problem.
+
+Specifically, triggering works like this:
+
+- New upstream release can trigger the workflow. (already implemented)
+- New commit to the branch can trigger the workflow. (needs to be enabled)
+- Adding `/propose-downstream` comment to any upstream issue. (already implemented)
+- Creating a new issue with some syntaxt.
+
+Updating the already created PRs:
+
+- Dist-git PR is closed and the workflow is retriggered (e.g. `/propose-downstream` comment). (already implemented)
+- Comment to the dist-git PR to re-create the PR.
+- Be able to push to the PR created by Packit.
+  - Needs support in Pagure.
+
+When reacting to comment, we now add :+1: to the comment so the user now we accepted the event.
+
+#### How to enhance the workflow?
+
+As a maintainer, I can influence the following:
+
+- Which branch is updated.
+- How the new changelog entry looks like.
+- What files are synced to dist-git.
+- Some command that is run to change the state of the dist-git.
+
+When there is a packit config available, we can use it. If not, the configuration is not possible.
+
+#### What this should done?
+
+1. Find the source for the new release and upload it to the lookaside cache if it is not already there.
+2. Update the version in specfile
+3. Create the new changelog entry. There are multiple options for this:
+
+- Changelog contains some generic text like `New upstream version X.Y.Z`
+- Changelog is generated from the git history.
+- Changelog is copied from the upstream (GitHub) release.
+- Changelog is generated using a custom command. (Some environment variables can be provided what can be used when the syntax of the first option can't be used.)
+
+### Implementation
+
+#### How we can get the event?
+
+If user install Packit GitHub application, we can get the event via webhook from the GitHub.
+If user does not have Packit installed in upstream, we can react on the fedora-messaging messages created by Upstream Release Monitoring. (Package can be added on https://release-monitoring.org/ .)
+
+#### How to implement it?
+
+We already have this implemented.
+
+What can be added is providing a feedback during the process:
+
+- providing a dashboard view for the job
```

Comment:
we could link to the Fedora developer dashboard: https://packager-dashboard.fedoraproject.org/

although they provide user-based view and not a package-based view